.cargo/config.toml

```diff
@@ -1,3 +1,6 @@
 [env]
 CFLAGS_wasm32_unknown_unknown = "-I wasm-sysroot"
 AR = 'llvm-ar'
```

Comment:
sidenote, I get an error with this set.

```
warning: tree-sitter@0.20.10: llvm-ar.exe: error: unknown option :

error: failed to run custom build command for `tree-sitter v0.20.10`

Caused by:
  process didn't exit successfully: `G:\workspace\Stationeers\ic10emu\target\debug\build\tree-sitter-ed09031b0f428758\build-script-build` (exit code: 1)

[snip]
  error occurred: Command "llvm-ar" "-out:G:\\workspace\\Stationeers\\ic10emu\\target\\debug\\build\\tree-sitter-ic10-f0d065e7cd7cda5c\\out\\libparser.a" "-nologo" "G:\\workspace\\Stationeers\\ic10emu\\target\\debug\\build\\tree-sitter-ic10-f0d065e7cd7cda5c\\out\\2e40c9e35e9506f4-parser.o" with args "" did not execute successfully (status code exit code: 1).
```

Reply:
This is mostly necessary for building on windows. if you have a environment with unix headers and gnu `ar` it can be droped

Reply:
welp, turns out whatever reason I needed it back at the start is now gone. it was something to do with how tree sitter was built at the time. Dropping it.